nzbtomedia/extractor/extractor.py

```diff
@@ -97,13 +97,17 @@ def extract(filePath, outputDestination):
     pwd = os.getcwd()  # Get our Present Working Directory
     os.chdir(outputDestination)  # Not all unpack commands accept full paths, so just extract into this directory
     devnull = open(os.devnull, 'w')
+    
```

Comment:
Can this info section be added inside the 
`if platform.system() == 'Windows':` section (line 12)

also, I guess we need to have a "default" info to pass into the subprocess command when using Posix...

Same for Linktastic... we only want to call subprocess.STARTUPINFO() when using Windows....